src/Microsoft.AspNet.Mvc.Razor/RazorViewEngineOptions.cs

```diff
@@ -65,5 +68,22 @@ public string Configuration
                 _configuration = value;
             }
         }
+
+        /// <summary>
+        /// Gets or sets the callback that is used to customize Razor compilation
```

Comment:
Should definitely add a comment stating that customizations here will not reflect in tooling. It'd normally go in remarks but users wouldn't be able to see it in IntelliSense.